keyboards/cybergear/macro25/keymaps/via/via-macro25.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "CyberGear Macro25 (mk-macro-01)",
+  "vendorId": "1209",
+  "productId": "69A1",
+  "lighting": "none",
+  "matrix": {
+    "rows": 2,
+    "cols": 5
+  },
+  "layouts": {
+    "keymap": [
+      [ "0,0", "0,1", "0,2", "0,3", "0,4" ],
+      [ "1,0", "1,1", "1,2", "1,3", "1,4" ]
+    ]
+  }
+}
```

Comment:
This json is for the VIA side over here https://github.com/the-via/keyboards/ It shouldn't be in QMK :)